docs/releases/3.1.txt

```diff
@@ -192,6 +192,11 @@ Minor features
 
 * ...
 
+Asynchronous
+~~~~~~~~~~~~
+
+* Added a system check to enforce async-safety environment on deployment.
+
```

Comment:
Actually looking at historical new checks we haven't added release notes for new system checks so I'm dropping this. Sorry!